### PR TITLE
adding Fn ascii to init command

### PR DESCRIPTION
**Pull request by carimura**

### CODE REVIEW

File: init.go [hunk 1]
```diff
@@ -126,6 +126,15 @@ func (a *initFnCmd) init(c *cli.Context) error {
 		}
 	}
 
+	const runHeader = `
```
Comment (treeder) [inline init.go]: Seems a bit much for a command like this... 🤷‍♀️   Lots of extra lines with no real value.

Comment (carimura) [reply init.go]: it's either there or when you install the CLI... Serverless does it at their init command. I think it fits. People in theory won't be fn init'ing all the time.

Comment (carimura) [reply init.go]: Ping. Value is branding and small touches. Can we merge?

**Comment by treeder** [comment]
Conflicts

**Comment by carimura** [comment]
fixed

**Pull request merged**